core/src/main/java/jenkins/install/InstallUtil.java

```diff
@@ -183,15 +183,15 @@ static void saveLastExecVersion(@Nonnull String version) {
     }
 
     static File getConfigFile() {
-        return new File(Jenkins.getActiveInstance().getRootDir(), "config.xml");
+        return new File(Jenkins.getInstance().getRootDir(), "config.xml");
     }
 
     static File getLastExecVersionFile() {
-        return new File(Jenkins.getActiveInstance().getRootDir(), ".last_exec_version");
+        return new File(Jenkins.getInstance().getRootDir(), "jenkins.install.InstallUtil.lastExecVersion");
```

Comment:
I agree with @kzantow though I'd like to use some convention clear and documented for this kind of marker-files.